services/static-badge/static-badge.service.js

```diff
@@ -3,13 +3,62 @@ import { BaseStaticService } from '../index.js'
 
 export default class StaticBadge extends BaseStaticService {
   static category = 'static'
-
   static route = {
     base: '',
     format: '(?::|badge/)((?:[^-]|--)*?)-?((?:[^-]|--)*)-((?:[^-.]|--)+)',
     capture: ['label', 'message', 'color'],
   }
 
+  static examples = [
+    {
+      title: 'Static Badge',
+      namedParams: { badgeContent: 'build-passing-brightgreen' },
```

Comment:
This param can't be called `:label-message-color`.
I went for `:badgeContent`, but it could be something else. `:labelMessageColor`? `:allTheThings`?

Reply:
I think `badgeContent` is fine, `labelMessageColor` or `badgeComponents` would also be fine with me

Reply:
I reckon let stick with `badgeContent`